kafka-connect-rethink/src/test/scala/com/datamountaineer/streamreactor/connect/rethink/config/TestReThinkConstants.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 Datamountaineer.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.datamountaineer.streamreactor.connect.rethink.config
+
+import com.datamountaineer.streamreactor.connect.rethink.TestBase
+
+/**
+  * The point of this test is to check that constants keys are not changed after the refactor of the code.
+  */
+class TestReThinkConstants extends TestBase {
+
+  // Constants
+  val RETHINK_HOST = "connect.rethink.sink.host"
+  val RETHINK_DB = "connect.rethink.db"
+  val RETHINK_PORT = "connect.rethink.sink.port"
+  val EXPORT_ROUTE_QUERY = "connect.rethink.kcql"
+  val ERROR_POLICY = "connect.rethink.error.policy"
+  val ERROR_RETRY_INTERVAL = "connect.rethink.retry.interval"
+  val NBR_OF_RETRIES = "connect.rethink.max.retries"
+  val BATCH_SIZE = "connect.rethink.batch.size"
+
+  "RETHINK_HOST should be equals to the same key in ReThinkSinkConfigConstants" in {
```

Comment:
shouldn't it be written as 'should be ->equal<-'?
or maybe:
"RETHINK_HOST should have the same key in ReThinkSinkConfigConstants"

edit: the same for the others =)